### ISSUE TITLE
Tentative timeline for full range of tones.

### ISSUE THREAD

**Issue by oahshtsua** [kepano/flexoki#84]
Would it be possible for you to provide a tentative date on when we might be getting a full range of tones for the colors? I am really loving the color scheme and would like to have the full range of tones for colors so that they can be used for things like diffs and highlights.

**Comment by kepano** [comment]
There is no timeline currently

**Comment by arranf** [comment]
@kepano I think this can be closed now 🎉

**Comment by kepano** [comment]
Yep! See [Flexoki 2.0](https://github.com/kepano/flexoki/releases/tag/2.0.0)
